build_uki.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
```

Comment:
i miss where this file is being consumed

Reply:
Currently a manual process to generate the images as I'm still working on it, order, retouches and also needs an extra path to generate the artifacts